app/index.js

```diff
@@ -10,7 +10,7 @@ import routes from './routes';
 import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import '../node_modules/toastr/build/toastr.min.css';
 import '../node_modules/griddle-react-bootstrap/dist/griddle-react-bootstrap.css';
-import '../ux/dev/css/styles.css';
+// import '../ux/dev/css/styles.css'; //ToDo: commenting MSUIX styles until fully integrated.
```

Comment:
Sorry @AlphaGit didn't get your point...
That's the reason: I commented with a ToDo note the import for the MSUIX styles so we use bootstrap for now temporary until the integration that we are working on, is fully merged.